src/catkin_pkg/topological_order.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, package, path):
         self.message_generator = message_generators[0] if message_generators else None
         # full includes direct build depends and recursive run_depends of these build_depends
         self.depends_for_topological_order = None
+        self.run_depends_for_topological_order = None
```

Comment:
This member should be renamed to `_recursive_run_depends_for_topological_order` to a) be private and b) contain "recursive" as part of the name.

Reply:
Member renamed.